systemtest/src/main/java/io/strimzi/systemtest/AbstractST.java

```diff
@@ -655,9 +655,11 @@ void verifyLabelsForRoleBindings(String clusterName, String appName) {
 
     /**
      * Wait till all pods in specific namespace being deleted and recreate testing environment in case of some pods cannot be deleted.
+     * This method is @Deprecated and it should be removed in the future
```

Comment:
It's more helpful to explain alternatives, or explain what's wrong with it.

Reply:
Good point. I added some additional info. We will deal with it as part of planed refactor in the following weeks.